operations/iuf/IUF.md

```diff
@@ -397,7 +398,16 @@ options:
   -cmrp CONCURRENT_MANAGEMENT_ROLLOUT_PERCENTAGE, --concurrent-management-rollout-percentage CONCURRENT_MANAGEMENT_ROLLOUT_PERCENTAGE
                         Limit the number of management nodes that roll out
                         concurrently based on the percentage specified. Must be an integer
-                        between 1-100. Defaults to 20 (percent).
+  -smc, --set-management-config  
```

Comment:
I believe this line was removed by accident. 
```suggestion
between 1-100. Defaults to 20 (percent).
-smc, --set-management-config
```

Reply:
Also remove the trailing whitespace on this line